stl/inc/yvals_core.h

```diff
@@ -1372,6 +1373,7 @@
 #define __cpp_lib_is_scoped_enum 202011L
 
 #ifdef __cpp_lib_concepts
+#define __cpp_lib_monadic_optional        202110L

```

Comment:
FYI, LWG-3621 proposes to increment `<optional>`'s main feature-test macro, instead of adding this separate macro. Although the LWG issue has not yet been accepted (even tentatively), @CaseyCarter notes that multiple people on the LWG mailing list have expressed support for this issue, and nobody has spoken out against it. Although it's slightly aggressive, I think it would be reasonable to go ahead and implement this issue's proposed resolution (as the downsides if it aren't accepted are minimal and recoverable).

Reply:
I noticed that [libstdc++'s commit message](https://gcc.gnu.org/pipermail/libstdc++/2021-October/053357.html) mentions this issue, but decided to hold off until either http://wg21.link/sd6 or libstdc++ makes the change.